### DISCUSSION TITLE
Cookie size too large error in client-cookie

### DISCUSSION THREAD

**Discussion by Sugavanam** [OpenIDC/mod_auth_openidc#1083] [General]
Hello Team,
    I'm using client-cookie & OIDCSessionCookieChunkSize together but for few application, I'm getting header size too large error while sending cookie. I was wondering is there any option in OIDC to reduce the size of the cookie by deciding what should be in it minimum to avoid this issue?

**Comment by zandbelt** [comment]
which version are you using?

**Reply by Sugavanam** [reply]
release 2.4.13.2

**Reply by zandbelt** [reply]
as the release notes show the cookie size has been reduced >= 2.4.14 and there is an option - when compiled iwth libjq - `OIDCFilterClaimsExpr` to filter the claims stored in the session

**Reply by Sugavanam** [reply]
Thanks! latest version along with BlacklistedClaims solved the issue
